drivers/pwm/pwm_rcar.c

```diff
@@ -0,0 +1,278 @@
+/*
+ * Copyright (c) 2022 IoT.bzh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT renesas_rcar_pwm
+
+#include <errno.h>
+#include <device.h>
+#include <devicetree.h>
+#include <soc.h>
+#include <drivers/pwm.h>
+#include <drivers/clock_control.h>
+#include <drivers/clock_control/rcar_clock_control.h>
+#include <drivers/pinctrl.h>
+
+#define LOG_LEVEL CONFIG_PWM_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(pwm_rcar);
+
+/* Helper macros for PWM */
+#define DEV_PWM_CFG(dev)        ((const struct pwm_rcar_cfg *)(dev)->config)
```

Comment:
Please drop these helper macros and use dev->data, dev->config directly

Reply:
Done :)